edx_notifications/server/web/utils.py

```diff
@@ -50,6 +50,9 @@ def get_notifications_widget_context(override_context=None):
             'user_notifications_all': (
                 '{base_url}?read=True&unread=True'
             ). format(base_url=reverse('edx_notifications.consumer.notifications')),
+            'user_notification_mark_read': (
+                '{base_url}'
+            ). format(base_url=reverse('edx_notifications.consumer.notifications.detail', args={'0'})),
```

Comment:
I would like to see if we can pass in args={''} so we don't get that trailing 0 that has to get fixed up in JavaScript.